### ISSUE TITLE
Add the tooltip entry to the translations.json (for Workflow tile)

### ISSUE THREAD

**Issue by adriatic** [monterey-framework/monterey#323]
The url is https://aurelia-ui-toolkits.gitbooks.io/monterey-user-manual/content/features/workflow_manager.html

### Also

Please write the brief definition of what should be the description of this feature in the above referenced tile and stuff it beneath this text

**Comment by JeroenVinke** [comment]
I wonder if we should hide this tile for now. I'd like to work on it some more so that it works better. The Run workflow would still work though, you just can't compose workflows yourself

**Comment by adriatic** [comment]
Yes, I agree with keeping it hidden (so do not update the translations.json). We could release this in a few weeks as a feature update (via Monterey auto-update, that needs to be kept in sync with User manual update). So, keep this item open as a reminder.
